### DISCUSSION TITLE
CapRover with Caddy?

### DISCUSSION THREAD

**Discussion by brandonzylstra** [caprover/caprover#1793] [Ideas]
Would it be possible to add Caddy to CapRover?

Would it make sense?
(Apart from personal preferences that anyone has for NGINX or Caddy, I mean is there something about CapRover that only makes sense with NGINX for some reason?)

It certainly seems like it would simplify CapRover since Caddy already has the SSL/LetsEncrypt feature built in, but obviously there are other considerations.

**Comment by githubsaturn** [comment]
CapRover has a strong dependency on nginx. All the custom configuration is built using nginx in mind.

Integrating Caddy is equivalent of rewriting a major part of CapRover.
